### PR TITLE
Added Sum of Matrix's Principal and Secondary Diagonal in CPP

### LINKED ISSUE #4458: Sum of Matrix's Principal and Secondary Diagonal in CPP
## 💥 Proposal

(A clear and concise description of what the proposal is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES.

### PR DESCRIPTION
**Pull request by RaiyanMahin**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES.

### Description

Given a N * N square matrix.
Find the sum of it's principal and secondary diagonal elements.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #4458

### CODE REVIEW

File: C-Plus-Plus/cp/matrix's_principal_seconday_diagonal_sum.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+Given a N * N square matrix.
+Find the sum of it's principal and secondary diagonal elements.
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/matrix's_principal_seconday_diagonal_sum.cpp]: Add bit more description

File: C-Plus-Plus/cp/matrix's_principal_seconday_diagonal_sum.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/*
+Given a N * N square matrix.
+Find the sum of it's principal and secondary diagonal elements.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+int ar[500][500];
+
+int get_principal_secondary_diagonal_sum(int N)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/matrix's_principal_seconday_diagonal_sum.cpp:12]: Add a one line function description above the function definition

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
@ankitaggarwal23  I made all the changes.

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Pull request merged**